storybook/src/components/Header/Header.stories.tsx

```diff
@@ -29,64 +65,62 @@ export const ForSubBrand: Story = {
 
 export const WithAppName: Story = {
   args: {
-    appName: 'Aan de Amsterdamse grachten',
+    appName: 'Onderzoek en Statistiek',
   },
 }
 
-export const WithLinks: Story = {
+export const WithMenu: Story = {
   args: {
-    links: (
-      <PageMenu alignEnd wrap={false}>
-        <PageMenu.Link href="#">Contact</PageMenu.Link>
-        <PageMenu.Link href="#">Mijn Amsterdam</PageMenu.Link>
-        <PageMenu.Link href="#" icon={SearchIcon}>
-          Zoeken
-        </PageMenu.Link>
+    menu: (
+      <PageMenu alignEnd>
+        <PageMenu.Item>
+          <PageMenu.Link href="#">Menu link 1</PageMenu.Link>
+        </PageMenu.Item>
+        <PageMenu.Item>
+          <PageMenu.Link href="#">Menu link 2</PageMenu.Link>
+        </PageMenu.Item>
+        <PageMenu.Item>
+          <PageMenu.Link href="#">Menu link 3</PageMenu.Link>
+        </PageMenu.Item>
       </PageMenu>
     ),
   },
 }
 
 export const WithMenuButton: Story = {
   args: {
-    menu: <button className="ams-header__menu-button">Menu</button>,
+    menu: <PageMenu alignEnd>{MenuButton}</PageMenu>,
   },
 }
 
-export const WithLinksAndMenuButton: Story = {
+export const WithMenuAndMenuButton: Story = {
   args: {
-    links: (
-      <PageMenu alignEnd wrap={false}>
-        <PageMenu.Link href="#">Contact</PageMenu.Link>
-        <PageMenu.Link href="#">Mijn Amsterdam</PageMenu.Link>
-        <PageMenu.Link href="#" icon={SearchIcon}>
-          Zoeken
-        </PageMenu.Link>
+    menu: (
+      <PageMenu alignEnd>
+        {SecondaryLinks}
+        {SearchButton}
+        {MenuButton}
       </PageMenu>
     ),
-    menu: <button className="ams-header__menu-button">Menu</button>,
   },
 }
 
-export const WithAppNameAndMenuButton: Story = {
+export const WithAppNameAndMenu: Story = {
   args: {
-    appName: 'Aan de Amsterdamse grachten',
-    menu: <button className="ams-header__menu-button">Menu</button>,
+    appName: 'Onderzoek en Statistiek',
+    menu: <PageMenu alignEnd>{MenuButton}</PageMenu>,
   },
 }
 
-export const WithAppNameLinksAndMenuButton: Story = {
+export const WithAppNameMenuAndMenuButton: Story = {
```

Comment:
Do we want to add a story here for the header + mega menu? It'd be nice to see the whole thing together, and I think I would look for it here.

Reply:
I want to make that the first entry in a new ‘Patterns’ section. Header + Mega Menu is a composition with some state: a pattern.

(The confusion may be that our Header component is just Logo + Heading + Page Menu, while visually the Main Menu and the Grid whitespace around it would conceptually belongs to the (application) header as well. I considered renaming Header to Masthead or Navigation Bar or something.)

Another reason is that this will give us room to present a couple of variations for larger and smaller sites, based on the navigation of some of our current sites.

Reply:
I think this depends on where we'll end up with our semantic definition of the Header. I think our Header contains our main navigation, which is comprised of the Page Menu and the Mega Menu. If we go with that definition, then the Mega Menu is part of the Header.